packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -224,7 +224,7 @@ async function handler(req: CustomRequest) {
 
   const evt: CalendarEvent = {
     title: bookingToDelete?.title,
-    type: (bookingToDelete?.eventType?.title as string) || bookingToDelete?.title,
+    type: (bookingToDelete?.eventType?.slug as string) || bookingToDelete?.title,
```

Comment:
Should the second part also be changed to slug like:
```
type: (bookingToDelete?.eventType?.slug as string) || bookingToDelete?.slug,
```

Reply:
Certainly, I missed onto that one.